protocol/message.go

```diff
@@ -146,13 +144,11 @@ func (m *Message) SetFilter(key, value string) {
 
 // IsExpired returns true if the message `Expires` field is set and the current time
 // has pasted the `Expires` time
-//
-// Checks are made using `Expires` field timezone
 func (m *Message) IsExpired() bool {
-	if m.Expires == nil {
-		return true
+	if m.Expires == 0 {
+		return false
 	}
-	return m.Expires != nil && m.Expires.Before(time.Now().In(m.Expires.Location()))
+	return m.Expires != 0 && m.Expires < time.Now().Unix()
```

Comment:
we already know at this point in code that m.Expires != 0
condition can be simplified